src/generic/toolbkg.cpp

```diff
@@ -336,6 +338,25 @@ bool wxToolbook::DeleteAllPages()
     return wxBookCtrlBase::DeleteAllPages();
 }
 
+bool wxToolbook::EnablePage(size_t page, bool enable)
+{
+    GetToolBar()->EnableTool(page + 1, enable);
+    if (!enable && GetSelection() == (int)page) {
```

Comment:
Minor style nitpick: could you please move the opening brace to its own line, to conform to wx coding style?